snap/snapcraft.yaml

```diff
@@ -1,106 +1,77 @@
+---
 name: couchdb
-version: 3.1.1
+version: 3.3.2
+type: app
+base: core22
 summary: Official Apache CouchDB snap - a clustering document oriented database
 description: |
   CouchDB is a database that completely embraces the web. Store your data with
   JSON documents. Access your documents and query your indexes with your web
   browser, via HTTP. Index, combine, and transform your documents with
   JavaScript.
-
-architectures:
-  - build-on: amd64
-    run-on: amd64
 assumes: [command-chain, common-data-dir]
-base: core20
 grade: stable
 confinement: strict
 
 parts:
-  add-repo:
-    plugin: nil
-    override-pull: |
-      apt-get update
-      apt-get upgrade -yy
-      apt-get install -y --no-install-recommends apt-transport-https \
-                                                gnupg ca-certificates
-      echo "deb https://apache.bintray.com/couchdb-deb focal main" | \
-           tee /etc/apt/sources.list.d/custom.list
-      apt-key adv --keyserver keyserver.ubuntu.com --recv-keys \
-                        8756C4F765C9AC3CB6B85D62379CE192D401AB61
-      apt-get update
   couchdb:
-    after: [add-repo]
-    plugin: dump
-    source: https://apache.bintray.com/couchdb-deb/pool/C/CouchDB/couchdb_3.1.1~focal_amd64.deb
-    source-type: deb
-    # because this doesn't use apt, we have to manually list all of our
-    # dependencies :(
+    plugin: make
+    source: https://dlcdn.apache.org/couchdb/source/3.3.2/apache-couchdb-3.3.2.tar.gz
+    source-type: tar
+    override-build: |
+            ./configure --disable-docs --spidermonkey-version 91
```

Comment:
Why did you disabled the inline docs?

Reply:
Speed of building and smaller snap size. Probably more of the former than the latter.